doc/source/development/contributing.rst

```diff
@@ -254,6 +257,29 @@ You'll need to have at least python3.5 installed on your system.
    python setup.py build_ext --inplace -j 4
    python -m pip install -e . --no-build-isolation
 
+**Windows**
+
+Below is a brief overview on how to set-up a virtual environment with Powershell
+under Windows. For details please refer to the \
+`official virtualenv user guide <https://virtualenv.pypa.io/en/stable/userguide/#activate-script>`__
+
+.. code-block:: none
+
+   # Create a virtual environment
+   # Use an ENV_DIR of your choice. We'll use ~\virtualenvs\pandas-dev
+   # Any parent directories should already exist
+   python -m venv ~\virtualenvs\pandas-dev
+
+   # Activate the virtualenv
+   ~\virtualenvs\pandas-dev\Scripts\Activate.ps1
+
+   # Install the build dependencies
+   python -m pip install -r requirements-dev.txt
+
+   # Build and install pandas
+   python setup.py build_ext --inplace -j 4
```

Comment:
not sure why we had `-j 4` initially, but I think it's better `-j auto`, so all cores are used.

Reply:
At least under Windows you need to use an integer for -j, 4 is a good value IMHO. 

![Capture](https://user-images.githubusercontent.com/3358056/67565120-f1da2700-f724-11e9-9ce4-0ff3b7fec2b5.PNG)